grails-app/views/inventory/browse.gsp

```diff
@@ -109,7 +109,9 @@
 														<td>
 															<g:link controller="inventoryItem" action="showStockCard" id="${searchResult?.product?.id}" style="color: ${searchResult?.product?.color}">
 																${searchResult?.product?.productCode}
-																${searchResult?.product?.name}
+																<span title="${searchResult?.product?.translatedName ? searchResult?.product?.name : null}">
```

Comment:
You could add a tagLib in product tagLibs for this. Plus I see, that you sometimes have product code, then you could add a boolean parameter to the tagLib and depending on that render it with code or not.